### PR TITLE
[IMP] inventory: full or partial packaging

### PR DESCRIPTION
**Pull request by Felicious**
## Summary
Implement @GillesMerckxOdoo's feedback for improving packaging doc

1. Move a tip admonition block explaining that packagings are useful when used with the Barcode app higher up the doc
2. Add section detailing the "Reserve only full/partial packagings" feature, in the "packaging" product category.
3. Fwport all the way to master

[Task](https://www.odoo.com/odoo/project/3835/tasks/3923319?cids=3)

### CODE REVIEW

**Review by pabr-odoo — APPROVED** [review]
@Felicious Everything looks good!

**Comment by Felicious** [comment]
Hi @odoo/us-doc-review ! Here's a very quick ~35 line PR for you :smile_cat:

File: content/applications/inventory_and_mrp/inventory/management/products/usage.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/products/usage.rst:225]:
```suggestion
expedites order fulfillment by allowing the immediate shipment of available items, while awaiting the
```

File: content/applications/inventory_and_mrp/inventory/management/products/usage.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/products/usage.rst:229]:
```suggestion
--> Product Categories`. Then, click :guilabel:`Create`, or select the desired product category. On
```

File: content/applications/inventory_and_mrp/inventory/management/products/usage.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/products/usage.rst:231]: Screenshot 2024-05-30 062337
On the 15.0 runbot I'm using for the review, I am not seeing a 'Reserve Packagings' field (in the Logistics section or anywhere else) on the Product Category form.
Is this a "runbot-related" issue? Does something else need to be enabled/configured before this field appears on a form? Either way, I think it might be worth looking into during your revisions, just to make sure everything is accurate. Thanks! 👍

File: content/applications/inventory_and_mrp/inventory/management/products/usage.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/products/usage.rst:238]:
```suggestion
   To better evaluate the options based on business needs, consider the following example:
```

**Review by ksc-odoo — CHANGES_REQUESTED** [review]
Hey, @Felicious -- just finished my Final Review on this succinct PR. Nice job. However, despite there only being a small handful of suggestions that require your attention, there *was* an instance where I wasn't seeing a field that you primarily focus on, so I think if you give that a quick look, and add any necessary clarification, it should be good to go. So, give that a review, and tag me again for another quick look. Thanks!

File: content/applications/inventory_and_mrp/inventory/management/products/usage.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/management/products/usage.rst:231]: Thanks for pointing this out! I'll add an "important" admonition block so the reader will remember to enable the relevant feture.

Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/management/products/usage.rst:231]:
```suggestion
the product category's form, in the :guilabel:`Logistics` section, :guilabel:`Reserve Packagings`
can be set to :guilabel:`Reserve Only Full Packagings` or :guilabel:`Reserve Partial Packagings`.

.. important::

Ensure the TODO: product packagings feature is enabled
```

**Comment by Felicious** [comment]
Thank you for catching the missing field, @ksc-odoo! I've added a note to remind users that the packaging feature must be enabled to see it. This PR is ready for your second review. Thanks for your hard work and time! I'm happy to address any other comments you have.

File: content/applications/inventory_and_mrp/inventory/management/products/usage.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/products/usage.rst:231]:
```suggestion
To configure packaging reservation methods, go to :menuselection:`Inventory app --> Configuration
--> Product Categories`. Then, click :guilabel:`Create`, or select the desired product category.

On the product category's form, in the :guilabel:`Logistics` section, :guilabel:`Reserve Packagings`
can be set to :guilabel:`Reserve Only Full Packagings` or :guilabel:`Reserve Partial Packagings`.
```

File: content/applications/inventory_and_mrp/inventory/management/products/usage.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/products/usage.rst:236]:
```suggestion
   To see the :guilabel:`Reserve Packaging` field, the :guilabel:`Product Packaging` feature **must** be
   enabled. To enable this feature, go to :menuselection:`Inventory app --> Configuration --> Settings`,
   scroll to the :guilabel:`Products` section, tick the :guilabel:`Product Packagings` checkbox, and click
   :guilabel:`Save`.
```

Comment (ksc-odoo) [reply content/applications/inventory_and_mrp/inventory/management/products/usage.rst:236]: Great job with the new admonition! Works perfectly now! Way to go 👍

File: content/applications/inventory_and_mrp/inventory/management/products/usage.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/management/products/usage.rst:253]:
```suggestion
   Conversely, when :guilabel:`Reserve Partial Packagings` is selected, twenty-two units are reserved for
   the order.
```

**Review by ksc-odoo — APPROVED** [review]
Great job with the necessary admonition and adjustments on this PR, @Felicious -- fantastic work! Only have a small handful of minor suggestions that require your attention. But, once you implement the necessary changes, feel free to tag this for Tech Review. Nicely done! 👍

**Comment by Felicious** [comment]
Hello @samueljlieber ! This ~50 line PR is ready for tech review, whenever you get the chance. Thank you!!

File: content/applications/inventory_and_mrp/inventory/management/products/usage.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/products/usage.rst:242]: Nitpick– no formatting in alt tags please :)
```suggestion
   :alt: Show Reserve Packagings field on the product categories page.
```

**Review by samueljlieber — APPROVED** [review]
Hi @Felicious, your changes look good to me, thanks for your work! Approving with one quick nitpick :)
..
@robodoo delegate=Felicious

**Comment by Felicious** [comment]
@robodoo r+

**Pull request closed without merging**
